src/constructs/cloudwatch/ec2-alarms.ts

```diff
@@ -62,23 +62,27 @@ export class Gu5xxPercentageAlarm extends GuAlarm {
 export class GuUnhealthyInstancesAlarm extends GuAlarm {
   constructor(scope: GuStack, props: GuUnhealthyInstancesAlarmProps) {
     const alarmName = `Unhealthy instances for ${props.app} in ${scope.stage}`;
-    const alarmDescription = `${props.app}'s instances have failed healthchecks several times over the last hour.
+
+    const period = Duration.minutes(5);
+    const evaluationPeriods = 12;
+    const evaluationInterval = Duration.minutes(period.toMinutes() * evaluationPeriods).toHumanString();
+
+    const alarmDescription = `${props.app}'s instances have failed healthchecks several times over the last ${evaluationInterval}.
       This typically results in the AutoScaling Group cycling instances and can lead to problems with deployment,
       scaling or handling traffic spikes.
 
       Check ${props.app}'s application logs or ssh onto an unhealthy instance in order to debug these problems.`;
-    const alarmProps = {
+    const alarmProps: GuAlarmProps = {
       ...props,
       alarmName: alarmName,
       alarmDescription: alarmDescription,
-      metric: props.targetGroup.metricUnhealthyHostCount(),
+      metric: props.targetGroup.metricUnhealthyHostCount().with({ period }),
```

Comment:
`period` at the top level is [deprecated](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-cloudwatch.AlarmProps.html#periodspan-classapi-icon-api-icon-deprecated-titlethis-api-element-is-deprecated-its-use-is-not-recommended%EF%B8%8Fspan), this change follows the advice in the deprecation warning.